### ISSUE TITLE
ci: make nightly fails on master

### ISSUE THREAD

**Issue by goller** [influxdata/platform#2044]
```
npm run build

> chronograf-ui@2.0.0 build /go/src/github.com/influxdata/platform/ui
> parcel build -d build --no-source-maps --log-level 2 src/index.html

make[1]: Leaving directory '/go/src/github.com/influxdata/platform/ui'
make[1]: Entering directory '/go/src/github.com/influxdata/platform/query'
make -C promql all
make[2]: Entering directory '/go/src/github.com/influxdata/platform/query/promql'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/go/src/github.com/influxdata/platform/query/promql'
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/query'
make[1]: Entering directory '/go/src/github.com/influxdata/platform/storage'
make -C reads all
make[2]: Entering directory '/go/src/github.com/influxdata/platform/storage/reads'
make -C datatypes all
make[3]: Entering directory '/go/src/github.com/influxdata/platform/storage/reads/datatypes'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/go/src/github.com/influxdata/platform/storage/reads/datatypes'
go generate -x
env GO111MODULE=on go run github.com/benbjohnson/tmpl -data=@array_cursor.gen.go.tmpldata array_cursor.gen.go.tmpl
package github.com/benbjohnson/tmpl: unknown import path "github.com/benbjohnson/tmpl": import lookup disabled by -mod=readonly
gen.go:3: running "env": exit status 1
Makefile:37: recipe for target 'array_cursor.gen.go' failed
make[2]: *** [array_cursor.gen.go] Error 1
make[2]: Leaving directory '/go/src/github.com/influxdata/platform/storage/reads'
Makefile:13: recipe for target 'reads' failed
make[1]: *** [reads] Error 2
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/storage'
make[1]: Entering directory '/go/src/github.com/influxdata/platform/task'
make -C backend all
make[2]: Entering directory '/go/src/github.com/influxdata/platform/task/backend'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/go/src/github.com/influxdata/platform/task/backend'
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/task'
env GO111MODULE=on go build -tags 'assets ' -o bin/linux/influx ./cmd/influx
# github.com/influxdata/platform/chronograf/canned
chronograf/canned/bin.go:20:11: undefined: AssetNames
chronograf/canned/bin.go:23:18: undefined: Asset
# github.com/influxdata/platform/chronograf/dist
chronograf/dist/dist.go:39:13: undefined: AssetInfo
chronograf/dist/dist.go:59:18: undefined: Asset
chronograf/dist/dist.go:74:11: undefined: Asset
chronograf/dist/dist.go:83:14: undefined: AssetDir
chronograf/dist/dist.go:84:14: undefined: AssetInfo
Makefile:64: recipe for target 'bin/linux/influx' failed
make: *** [bin/linux/influx] Error 2
Exited with code 2
```

@nathanielc looks like two problems: array_cursor_gen.go is not able to be be built and chronograf assets (in the dist package) are not built.

**Comment by mark-rushakoff** [comment]
The deploy stage is still failing, but for new reasons now:

https://circleci.com/gh/influxdata/platform/10349

```
bindata: Failed to stat input path '../../ui/build': lstat ../../ui/build: no such file or directory
exit status 1
dist.go:3: running "env": exit status 1
Makefile:24: recipe for target 'dist_gen.go' failed
make[2]: *** [dist_gen.go] Error 1
make[2]: Leaving directory '/go/src/github.com/influxdata/platform/chronograf/dist'
Makefile:13: recipe for target 'dist' failed
make[1]: *** [dist] Error 2
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/chronograf'
make[1]: Entering directory '/go/src/github.com/influxdata/platform/http'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/http'
make[1]: Entering directory '/go/src/github.com/influxdata/platform/ui'

<snip>

stringer -type=readState -trimprefix=state
gen.go:7: running "stringer": exec: "stringer": executable file not found in $PATH
Makefile:37: recipe for target 'array_cursor.gen.go' failed
make[2]: *** [array_cursor.gen.go] Error 1
make[2]: Leaving directory '/go/src/github.com/influxdata/platform/storage/reads'
Makefile:13: recipe for target 'reads' failed
make[1]: *** [reads] Error 2
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/storage'
make[1]: Entering directory '/go/src/github.com/influxdata/platform/task'
make -C backend all
make[2]: Entering directory '/go/src/github.com/influxdata/platform/task/backend'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/go/src/github.com/influxdata/platform/task/backend'
make[1]: Leaving directory '/go/src/github.com/influxdata/platform/task'

<snip>

# github.com/influxdata/platform/chronograf/canned
chronograf/canned/bin.go:20:11: undefined: AssetNames
chronograf/canned/bin.go:23:18: undefined: Asset
# github.com/influxdata/platform/chronograf/dist
chronograf/dist/dist.go:39:13: undefined: AssetInfo
chronograf/dist/dist.go:59:18: undefined: Asset
chronograf/dist/dist.go:74:11: undefined: Asset
chronograf/dist/dist.go:83:14: undefined: AssetDir
chronograf/dist/dist.go:84:14: undefined: AssetInfo
Makefile:64: recipe for target 'bin/linux/influx' failed
make: *** [bin/linux/influx] Error 2
Exited with code 2
```

We also aren't restoring the module cache properly, but I'm going to track that in a separate issue.

**Comment by mark-rushakoff** [comment]
The module cache issue I mentioned in my previous comment was fixed in #2057.
